src/libraries/System.Private.Uri/tests/UnitTests/IriEscapeUnescapeTest.cs

```diff
@@ -223,214 +223,44 @@ private string EscapeUnescapeTestComponent(string uriInput, UriComponents compon
             return ret;
         }
 
-        [Fact]
-        public void Iri_MatchUTF8Sequence_valid_utf_followed_by_invalid_utf_5byte()
-        {
-            byte[] bytes = new byte[] { 0xF4, 0x80, 0x80, 0xBA, 0xFD, 0x80, 0x80, 0xBA, 0xCD };
-            MatchUTF8SequenceTest(bytes, bytes.Length);
-        }
```

Comment:
Removed these UTF8SequenceTests as the method doesn't exist anymore. Will add similar edge-case utf8 tests targeting PercentEncodingHelper to this PR later